frontend/src/shared/helpers/attribute.helpers.ts

```diff
@@ -0,0 +1,30 @@
+import { CrowdIntegrations } from '@/integrations/integrations-config';
+import { isEqual } from 'lodash';
+
+export const getAttributeSources = (attribute: Record<string, string>): string[] => {
+  const defaultValue: string | undefined = attribute.default;
```

Comment:
I just created a global attribute and the member form page broke on the attributes section.
For global attributes we don't want to show this symbol.
Can you protect in these cases?

Reply:
found the issue and fixed it